blueprints/controller/files/__root__/__path__/__name__.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Controller definition for the <%= dasherizedModuleName %> controller
```

Comment:
Maybe it is just me but what do these comments really add at the top of the files? It seems to me if you understand the basics of Ember and where files are located these provide nothing new.